backend/plugins/tourism_api/src/modules/ota/db/models/Hotels.ts

```
@@ -0,0 +1,57 @@
+      if (!hotel) {
+        throw new Error('Hotel not found');
+      }
+
+      hotel.isPublished = hotel.isPublished ? false : true;
+      const updatedHotel = await models.Hotels.findOneAndUpdate(
+        { _id },
+        { $set: hotel },
+      );
+      return updatedHotel;
+    };
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)